packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -108,78 +117,6 @@ function decryptSpace(space) {
   return Promise.resolve(new Error('Space cannot be decrypted'));
 }
 
-
-/**
- * Fetches a list of spaces with options
- *
- * @export
- * @param {Object} sparkInstance
- * @param {Object} options
- * @returns {Function} thunk
- */
-export function getSpaces(sparkInstance, options = {}) {
-  const listOptions = Object.assign({
-    uuidEntryFormat: true,
-    personRefresh: true,
-    isActive: true,
-    lastViewableActivityOnly: true,
-    participantAckFilter: 'all',
-    deferDecrypt: false,
-    allFavorite: true,
-    participantsLimit: -1
-  }, options);
-  const {deferDecrypt} = options;
-
-  return (dispatch) => sparkInstance.internal.conversation
-    .list(listOptions)
-    .then((items) => {
-      const users = [];
-      const activities = [];
-
-      const spaces = items.map((space) => {
-        let decryptPromise;
-
-        // Start decryption
-        if (deferDecrypt && space.decrypt) {
-          decryptPromise = decryptSpace(space)
-            .then((decryptedSpace) => {
-              if (decryptedSpace) {
-                const s = Object.assign({}, decryptedSpace, {isDecrypting: false});
-                const decryptedActivities = decryptedSpace.activities.items;
-                const participants = decryptedSpace.participants.items;
-                const activityActors = activities.map((a) => a.actor);
-
-                dispatch(storeUsers([...participants, ...activityActors]));
```

Comment:
@akoushke this is where the old code was storing users and activities. When you had the console open you would see a flood of the actions for store user and store activity come across as spaces decoded. Realized this was extra processing that wasn't needed for the recents widget at all with the new design.

Reply:
got it. thanks!